src/lzcv/generator/agri2030/energy_source.py

```diff
@@ -0,0 +1,406 @@
+# pyright: strict
```

Comment:
I think, this is an old file, which you accidently commited

Reply:
Same was true for energy_demand.py in the same file.  Thanks again.